script/config.ts

```diff
@@ -6,4 +6,15 @@ export const imageMaxLogoSizeKb =  100;
 export const foldersRootdirAllowedFiles: string[] =  [".github", "blockchains", "dapps", "media", "node_modules", "script-old", "script", "test", ".gitignore", "azure-pipelines.yml", "jest.config.js", "LICENSE", "package-lock.json", "package.json", "README.md", ".git", "dangerfile.ts", "Gemfile", "Gemfile.lock", ".eslintignore", ".eslintrc.js"];
 export const binanceUrlTokenAssets =  "https://explorer.binance.org/api/v1/assets?page=1&rows=1000";
 export const binanceDexURL = 'https://dex-atlantic.binance.org/api'
-export const assetsURL = 'https://raw.githubusercontent.com/trustwallet/assets/master'
\ No newline at end of file
+export const assetsURL = 'https://raw.githubusercontent.com/trustwallet/assets/master'
+
+export const PancakeSwap_TradingPairsUrl = "https://api.bscgraph.org/subgraphs/name/wowswap";
+export const PancakeSwap_TradingPairsQuery = "query pairs {\\n  pairs(first: 400, orderBy: reserveUSD, orderDirection: desc) {\\n id\\n reserveUSD\\n trackedReserveETH\\n volumeUSD\\n txCount \\n   untrackedVolumeUSD\\n __typename\\n token0 {\\n id\\n symbol\\n name\\n decimals\\n __typename\\n }\\n token1 {\\n id\\n symbol\\n name\\n decimals\\n __typename\\n }\\n }\\n}\\n";
```

Comment:
can you use formatter for this query? so we can easily adjust

Reply:
Good idea, done